CONTRIBUTING.md

```diff
@@ -1,16 +1,28 @@
+# Contributing to Riot.js
 
-### Coding style
+If you’d like to contribute a feature or bug fix, you can [fork](https://help.github.com/articles/fork-a-repo/) Riot, commit your changes, & [send a pull request](https://help.github.com/articles/using-pull-requests/).
+Please make sure to [search the issue tracker](https://github.com/mutt/riotjs/issues) first.
 
-The most important:
+### Development Process
 
-- Leave out semicolons
-- Use single quotes and not double quotes
-- Prefer `==` over `===` unless it's a must
-- Spacing around brackets: `if (foo) {` instead of `if(foo){`
+`master` is considered safe and should be identical to the last release tag. When submitting pull requests please use the `dev` branch.
```

Comment:
I'm assuming that this is what you meant by "Use `dev` branch" in the original guidelines.